8.1/Dockerfile

```diff
@@ -19,6 +20,11 @@ RUN gpg --keyserver ha.pool.sks-keyservers.net --recv-keys E3036906AD9F30807351F
 RUN docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr \
 	&& docker-php-ext-install gd intl mbstring mcrypt mysql pdo_mysql pdo_pgsql pgsql zip
 
+# PECL extensions
+RUN pecl install APCu-beta && echo extension=apcu.so > /usr/local/etc/php/conf.d/apcu.ini \
+	&& pecl install redis && echo extension=redis.so > /usr/local/etc/php/conf.d/redis.ini \
+	&& pecl install memcached && echo extension=memcached.so > /usr/local/etc/php/conf.d/memcached.ini
```

Comment:
Should swap the echoes for `docker-php-ext-enable apcu redis memcached` ([docker-php-ext-enable](https://github.com/docker-library/php/blob/757528222b84cd8204d5fe54dba31cb804b26b70/docker-php-ext-enable)).